presto-main/src/main/java/com/facebook/presto/sql/planner/Plan.java

```diff
@@ -54,4 +61,14 @@ public PlanNode getRoot()
     {
         return planNodeCosts;
     }
+
+    private List<PlanNode> getPlanNodes(PlanNode root)
+    {
+        ImmutableList.Builder<PlanNode> planNodes = ImmutableList.builder();
+        planNodes.add(root);
+        for (PlanNode source : root.getSources()) {
+            planNodes.addAll(getPlanNodes(source));
+        }
+        return planNodes.build();
```

Comment:
Actually I had to use:
```
searchFrom(root, Lookup.from(groupReference -> groupReference)).findAll();
```
As this is used (at least during tests) when `GroupReferences` are not yet resolved.